docs/concepts/ipfs-implementations

```diff
@@ -0,0 +1,14 @@
+---
+title: IPFS Implementations
+legacyUrl: ???
```

Comment:
This variable isn't really used anymore. It was only used during the migration from the super old site to VuePress (cira 2020). You can delete this line.